doc/src/explore/sui-explorer.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Using the Sui Explorer
+---
+
+# Using Sui Explorer
+The [Sui Explorer](https://explorer.devnet.sui.io/) lets you view data about transactions and activity on the Sui network. In addition to viewing activity on the network, you can use the Explorer to:
+ * View up-to-date information about the activity and metrics on the Sui network.
+ * Look up, verify, and track your assets and contracts.
+ * Utilize fast, reliable, and transparent debugging and auditing data to help identify and resolve issues. 
+ * Get go-to-definition support for all smart contracts, referred to as packages in Sui.
+ * View validators and geographic locations of full nodes that are currently active.
+
+## Supported Browsers
+The Sui Explorer supports the following browsers:
+ * Chrome
+ * Firefox
+ * Safari
+ * Edge
+
+## Choose a network
+When you start Sui Explorer, it displays the transactions for the Devnet network by default. You can also use the Explorer to view data for a local network running in your environment or a custom RPC endpoint URL.
+
+
+## Finding your transaction
+You can search for the transactions using an address, object ID, or transaction ID. For example, you can search for your wallet address to confirm a transaction or view additional details about a transaction you’ve approved. See [Sui Wallet](../explore/wallet-browser.md) to learn how to create a wallet).  
+
+**To search for a transaction made using the Sui wallet**
```

Comment:
Capitalize "Sui Wallet" in all uses since it's a proper noun (formal name of product).

And why is this intro text bold? Should it end with a colon for consistency with our other docs? Or any punctuation whatsoever?